src/Task.js

```diff
@@ -57,7 +57,6 @@ class Task {
 
             taskInfo.parameters.forEach((param) => {
               parameters[param.name] = param;
-              delete parameters[param.name].name;
```

Comment:
As a user of a TaskParameter object, e.g. as a webcomponent prop, it is SUPER inconvenient not to have the name in the object itself.